sympy/simplify/combsimp.py

```diff
@@ -217,6 +219,232 @@ def explicate(p):
                 denom_gammas.extend(l)
             elif isg is False:
                 denom_others.extend(l)
+        # =========== level >= 2 work: factor absorbtion =========
+        if level >= 2:
```

Comment:
When I removed If condition after line 223 i.e the whole if condition with its condition , it gave
![screen shot 2017-02-19 at 11 21 10 pm](https://cloud.githubusercontent.com/assets/19210197/23104783/57b25058-f6fa-11e6-9078-198ce6b86518.png)